src/chain_spec.rs

```diff
@@ -776,6 +776,10 @@ fn altair_genesis(
 		},
 		ethereum: Default::default(),
 		evm: Default::default(),
+		liquidity_rewards_base: altair_runtime::LiquidityRewardsBaseConfig {
```

Comment:
I think this will break if `testnet_runtime` isn't specified, since then this wouldn't exist in the Genesis?

I'd rather remove that feature gate anyway - making algol more-different from Altair is just going to be even more of a maintenance headache. Since we intend to land this feature in the Centrifuge runtime Soon:tm:, I think we should just add it to Altair properly.

Reply:
Oh, yup. The test failure is exactly the compile error I thought would happen

Reply:
Done